azurerm/resource_arm_virtual_machine_scale_set.go

```diff
@@ -393,24 +394,34 @@ func resourceArmVirtualMachineScaleSet() *schema.Resource {
 				MaxItems: 1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
```

Comment:
isn't it possible to change the image on a VMSS (and then either roll the VMSS instances automatically or manually)? As such, does this want to be without the `ForceNew`?

(same question for Publisher/Offer/Sku/Version)